main.go

```diff
@@ -19,51 +20,38 @@ const (
 )
 
 var (
-	username = kingpin.Flag("username", "Username").String()
-	password = kingpin.Flag("password", "Password").String()
-	endpoint = kingpin.Flag("url", "URL to proxy service to, which includes the port.").Short('u').String()
-	port     = kingpin.Flag("port", "Port to expose the service to the host").Default("8080").Short('p').Int()
+	PROXY_APP_ENDPOINT string
+	PROXY_APP_USERNAME string
+	PROXY_APP_PASSWORD string
+	PROXY_APP_PORT string
 )
 
-func main() {
-
-	kingpin.Parse()
-
+func init() {
```

Comment:
I would perfer to see a `main()` and `run()` approach if you want to initialize the Skpr client outside of `main()`.

Reply:
Done - `Run()` is exported for external use.